### PR TITLE
Hex config

### PR DESCRIPTION
**Pull request by misaelp5**

### CODE REVIEW

File: lib/papelillo/mailer_list.ex [hunk 1]
```diff
@@ -59,46 +59,111 @@ defmodule Papelillo.MailerList do
...
+  Address must be a string without blank spaces and special characters, the domain of the email address is taken from the config.exs definition.
+  """
   def create(name, description, address, config) do
     provider = Keyword.fetch!(config, :provider)
 
-    address = address <> "@" <> Keyword.fetch!(config, :domain)
+    address =
+      Keyword.fetch(config, :domain)
+      |> case do
+        {:ok, domain} -> address <> "@" <> domain
```
Comment (bulld0zer) [inline lib/papelillo/mailer_list.ex]: I think this looks better
```suggestion
        {:ok, domain} -> "#{address}@#{domain}"
```

**Review by benitezhm — DISMISSED** [review]
looks good to me, just add the docs for this new feature about the domain

File: lib/papelillo/mailer_list.ex [hunk 2]
```diff
@@ -175,4 +206,27 @@ defmodule Papelillo.MailerList do
...
+
+  def validate_email(email) do
+    if EmailChecker.Check.Format.valid?(email) do
+      email
+    else
+      raise "#{email} has not valid email format"
+    end
+  end
+
+  def validate_email(email, config) do
```
Comment (misaelp5) [inline lib/papelillo/mailer_list.ex:218]: @benitezhm @bulld0zer this is the new part, you don't need to not-config "domain" in config files, only you need to send a valid email if you want to use another domain besides of default

File: lib/papelillo/mailer_list.ex [hunk 3]
```diff
@@ -175,4 +206,27 @@ defmodule Papelillo.MailerList do
...
+    if EmailChecker.Check.Format.valid?(email) do
+      email
+    else
+      email =
+        Keyword.fetch(config, :domain)
+        |> case do
+          {:ok, domain} -> email <> "@" <> domain
```
Comment (bulld0zer) [inline lib/papelillo/mailer_list.ex]:
```suggestion
          {:ok, domain} -> "#{email}@#{domain}"
```

**Review by bulld0zer — DISMISSED** [review]

**Review by bulld0zer — APPROVED** [review]

**Pull request merged**